content/references/rippled-api/transaction-formats/pseudo-transaction-types/unlmodify.md

```diff
@@ -0,0 +1,50 @@
+---
+html: unlmodify.html
+funnel: Build
+doc_type: References
+supercategory: rippled API
+category: Transaction Formats
+subcategory: Pseudo-Transaction Types
+status: not_enabled
+blurb: Change the list of trusted validators currently considered offline.
+---
+# UNLModify
+
+_(Requires the [NegativeUNL amendment]() :not_enabled:)_
+
+A `UNLModify` [pseudo-transaction](pseudo-transaction-types.html) marks a change to the [Negative UNL](negative-unl.html), indicating that a trusted validator has gone offline or come back online.
+
+**Note:** You cannot send a pseudo-transaction, but you may find one when processing ledgers.
```

Comment:
do you mean you cannot send a pseudo-transaction while modifying a Negative UNL, or you cannot send a pseudo-transaction when processing ledgers? i might be confused because i don't have enough context about how XRP works, so maybe a dev would already know this

Reply:
You cannot send any pseudo-transaction, ever. They get directly injected into proposed ledgers as part of special consensus rules. Therefore, they appear in ledger data, but are not something that any one can "send" in the same sense as a normal transaction.